content/blog/using-graphql-with-the-filesystem.md

```diff
@@ -15,15 +15,32 @@ author: Jeff See
 
 ---
 
+---
+title: Limitations of using the filesystem for web content and how Tina's GraphQL API solves this
```

Comment:
Tina's GraphQL API brings data integrity and solid relationships to Git-based content management